src/cli.ts

```diff
@@ -162,12 +162,16 @@ export function getCli() {
 
       loadContract(argv.source, argv.network, address, argv.outdir, argv.verbose);
     })
-    .command('verify <apiKey> <contract>', 'Verify a given contract on Etherscan', (yargs) => {
+    .command('verify <apiKey> <address> <contract>', 'Verify a given contract on Etherscan', (yargs) => {
```

Comment:
Is this so we don't rely on `networks`? In that case can/should we drop `contract`?

Reply:
Contract is the name of the contract, so it's basically `verify API_KEY 0x... CErc20 ...args`

Reply:
Oh so instead of `--source` then?

Reply:
I think we don't need both `contract` and `--source` if we have the address. At least prior to this `contract` was the one that was used to resolve the address and `--source` was for the code lookup

Reply:
Source was removed in this change.

Reply:
https://github.com/compound-finance/saddle/pull/29/files/c2082160edd8c7c7d8b230c23cecccb133b89e71#diff-cd02b9e0b731a6a1dedefed142f4419bL180